content/kms-plugins.md

```diff
@@ -0,0 +1,39 @@
++++
+title = "KMS Plugins for Sigstore"
+author = "Ramon Petgrave (@ramonpetgrave64)"
+date = "2025-04-07"
+draft = false
+type = "post"
+tags = ["sigstore","kms", "plugin"]
++++
+
+Cosign and private deployments of Fulcio and Rekor can use a KMS-managed key for signing artifacts. We currently have built-in support for AWS, Azure, Google Cloud Platform, and Hashicorp Vault KMSs. This has been a challenge for customers that require alternative or custom KMS solutions.
+
+To enable such use-cases, we have implemented a new [plugin system](https://github.com/sigstore/sigstore/tree/main/pkg/signature/kms/cliplugin) for alternate KMS providers. Organizations can independently and privately develop & distribute their plugins without needing downstream updates to libraries to support additional KMS providers as build-time dependencies. You won't have to maintain your own fork of Cosign just to use other KMS providers.
+
+[Alibaba KMS](https://github.com/mozillazg/sigstore-kms-alibabakms) support is now possible thanks to a new plugin by community member @mozillazg.
+
```

Comment:
@zosocanuck would you like a mention for the [venafi plugin](https://github.com/Venafi/sigstore-kms-venafi)?